Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -931,7 +933,7 @@ def list_endpoint_security_template_command(client: Client) -> CommandResults:
     return command_results
 
 
-def start_endpoint_security_assessment_command(client: Client, template_id: str, agent_name: str,
+def start_endpoint_security_assessment_command(client: Client, template_id: str, agent_name: str,agent_Profile_Name: str,
```

Comment:
In the yml file it is called `agent_profile_name` and here `agent_Profile_Name`, does the command manage to run like this?
Also, the argument is optional, so what happens if the command is run without providing this arg?

Reply:
will change it for just in case